src/igvfd/schemas/measurement_set.json

```diff
@@ -2,7 +2,7 @@
     "title": "Measurement Set",
     "$id": "/profiles/measurement_set.json",
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "description": "A file set for a distinct measurement, such as the sequencing of a genomic library generated by an experiment performed on a cell line. A measurement set contains the raw data files generated by the measurement.",
+    "description": "Measurement Set is a file-set that hosts raw data files (e.g. FASTQs) that result from sequencing of a library prepared from the nucleic acids of the sample that is the main target of an assay. For example sequencing of accessible regions in the genome, or sequencing of the transcriptome of the sample. The assay can either be bulk or single cell type. The sample specific raw sequencing results will be caputured in the Measurement Set. The files in the Measurement Sets are specific to the sample being investigated. Note: see auxiliary set for files that are not direct result of sequencing the sample under investigation.",
```

Comment:
* Same comment about schema capitalization and no hyphen in "file set"
* Typo "caputured" instead of "captured"
* Can remove "Note:" so it just says "See auxiliary sets for files that...". In the same line, I would add "the" before direct result, "are not the direct result of sequencing..."